loader/internal/loader/loader.go

```diff
@@ -534,9 +534,8 @@ func buildModelIDForGGUF(modelID, ggufModelFilePath string) string {
 
 func toLLMarinerModelID(id string) string {
 	// HuggingFace uses '/' as a separator, but Ollama does not accept. Use '-' instead for now.
-	// Ollama uses ':' as a separator, it cannot be used for bucket name. Use '-' instead.
 	// TODO(kenji): Revisit this.
-	return strings.ReplaceAll(strings.ReplaceAll(id, "/", "-"), ":", "-")
+	return strings.ReplaceAll(id, "/", "-")
```

Comment:
Just wanted to check if there is any backward compatibility issue here.

If a user already has an Ollama, this change will trigger redownload?

Reply:
I guess this doesn't impact existing ones since most of them don't use `:`.

Reply:
ah anyway this is a very recent change..

Reply:
Yes, the model will be re-downloaded. Also, no component still does not support the ollama type, so this change would not affect users.

Reply:
Sounds good!